pkg/csi_driver/controller.go

```diff
@@ -394,9 +405,15 @@ func (s *controllerServer) generateNewFileInstance(name string, capBytes int64,
 			}
 		case paramNetwork:
 			network = v
+		case paramConnectMode:
+			connectMode = v
+			if connectMode != defaultConnectMode && connectMode != privateConnectMode {
+				return nil, fmt.Errorf("connect mode can only be one of %v or %v", defaultConnectMode, privateConnectMode)
```

Comment:
there is a "CONNECT_MODE_UNSPECIFIED". Is this a default value? https://cloud.google.com/filestore/docs/reference/rest/v1/projects.locations.instances#ConnectMode

Reply:
According to the same page it says "If not provided, the connect mode defaults to DIRECT_PEERING."

Reply:
so we should also support CONNECT_MODE_UNSPECIFIED as an acceptable string ?

Reply:
No we should not, checked with Filestore team and the "FOO_UNSPECIFIED" is just a proto enum specificity